x/confidentialtransfers/types/keys.go

```diff
@@ -19,10 +19,15 @@ const (
 
 	// QuerierRoute defines the module's query routing key
 	QuerierRoute = ModuleName
+
+	DefaultEnableFeature           = true
```

Comment:
These constants may belong to params.go file.

Reply:
Moved to params.go file